vendor/reqwest.min.js

```diff
@@ -1,522 +0,0 @@
-/*!
```

Comment:
yes, since we decided to :fire: nojquery/noleaflet it could be removed, see 6c998a9a310dfc669e2dd6354a5aa26806eb55cd